Annotato/Annotato/Persistence/LocalPersistenceManager/LocalPersistenceManager.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import CoreData
+
+struct LocalPersistenceManager {
+    static let sharedInstance = LocalPersistenceManager()
+    private static let containerName = "AnnotatoLocal"
+
+    private let container: NSPersistentContainer
+
+    private init() {
+        container = NSPersistentContainer(name: LocalPersistenceManager.containerName)
+        container.loadPersistentStores { _, error in
+            if let error = error {
+                AnnotatoLogger.error("Error loading Core Data: \(error)",
```

Comment:
By right we should crash the application if core data doesn't load right? This is because for both online/offline use we need to make sure core data is running properly. Should we `fatalError` instead?